lib/mercadopago/client.rb

```diff
@@ -142,6 +144,27 @@ def search(search_hash)
     end
 
     #
+    # Performs a generic GET to the given URL
+    #
+    # - url: the URL to request
+    #
+    def get(url)
+      MercadoPago::Request.wrap_get("#{url}?access_token=#{@access_token}")
```

Comment:
Done, I had to change the way default headers were set because if I'd set headers as `{}` in `Client.get` then `wrap_get` would not set the default headers, same for post.

Also, now it supports adding custom query strings.